src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,68 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
our email field have to be unique - use @Column(unique = true) with email or @UniqueConstraint in general table annotation

Reply:
fix that and you are all good

Reply:
> fix that and you are all good

Already fixed in @Table annotation